installation/routines/customize_options.sh

```diff
@@ -49,51 +49,101 @@ Do you want to disable IPv6? [Y/n]"
 }
 
 _option_autohotspot() {
-  # ENABLE_AUTOHOTSPOT
-  clear_c
-  print_c "---------------------- AUTOHOTSPOT ----------------------
+    # ENABLE_AUTOHOTSPOT
+    clear_c
+    print_c "---------------------- AUTOHOTSPOT ----------------------
 
 When enabled, this service spins up a WiFi hotspot
 when the Phoniebox is unable to connect to a known
 WiFi. This way you can still access it.
 
 Do you want to enable an Autohotpot? [y/N]"
-  read -r response
-  case "$response" in
-    [yY][eE][sS]|[yY])
-      ENABLE_AUTOHOTSPOT=true
-      ;;
-    *)
-      ;;
-  esac
-
-  if [ "$ENABLE_AUTOHOTSPOT" = true ]; then
-      print_c "Do you want to set a custom Password? (default: ${AUTOHOTSPOT_PASSWORD}) [y/N] "
-      read -r response_pw_q
-      case "$response_pw_q" in
+    read -r response
+    case "$response" in
         [yY][eE][sS]|[yY])
-          while [ $(echo ${response_pw}|wc -m) -lt 8 ]
-          do
-              print_c "Please type the new password (at least 8 character)."
-              read -r response_pw
-          done
-          AUTOHOTSPOT_PASSWORD="${response_pw}"
-          ;;
+            ENABLE_AUTOHOTSPOT=true
+            ;;
         *)
-          ;;
-      esac
+            ;;
+    esac
+
+    if [ "$ENABLE_AUTOHOTSPOT" = true ]; then
+        local response_autohotspot
+        while [[ $response_autohotspot != "n" ]]
+        do
+            print_c "
+--- Current configuration for Autohotpot
+SSID              : $AUTOHOTSPOT_SSID
+Password          : $AUTOHOTSPOT_PASSWORD
+IP                : $AUTOHOTSPOT_IP
+WiFi Country Code : $AUTOHOTSPOT_COUNTRYCODE
+Do you want to change this values? [y/N]"
+            read -r response_autohotspot
+            case "$response_autohotspot" in
+                [yY][eE][sS]|[yY])
+                    local response_ssid=""
+                    local response_ssid_length=0
+                    while [[ $response_ssid_length -lt 1 || $response_ssid_length -gt 32 ]]
+                    do
+                        print_c "Please type the hotspot ssid (must be between 1 and 32 characters long):"
+                        read -r response_ssid
+                        response_ssid_length=$(get_string_length ${response_ssid})
+                    done
+
+                    local response_pw=""
+                    local response_pw_length=0
+                    while [[ $response_pw_length -lt 8 || $response_pw_length -gt 63 ]]
+                    do
+                        print_c "Please type the new password (must be between 8 and 63 characters long):"
+                        read -r response_pw
+                        response_pw_length=$(get_string_length ${response_pw})
+                    done
+
+                    local response_ip=""
+                    while [[ ! "$response_ip" =~ ^((25[0-5]|(2[0-4]|1[0-9]|[1-9]|)[0-9])\.){3}((25[0-5]|(2[0-4]|1[0-9]|[1-9]|)[0-9]))$ ]]
```

Comment:
Should we really let people decide about the IP? Most of them don't have the knowledge I assume.

I would keep it as default.